docs/features/wait/health.md

```diff
@@ -0,0 +1,13 @@
+# Health Wait strategy
+
+The health wait strategy will check the container is in the healthy state, being able to set the following conditions:
+
+- the startupTimeout to be used, default is 60 seconds.
```

Comment:
```suggestion
- the startup timeout to be used in seconds, default is 60 seconds.
```
Since `exit timeout` hat a space, I thought this is more consistent.